src/store/mutations/mutations-model.ts

```diff
@@ -137,3 +137,7 @@ export const mutateHaveChanges = (state: any, haveChanges: boolean) => {
 export const mutateEntityType = (state: any, entityType) => {
   state.stateModel.entityType = entityType
 }
+
+export const mutateValidateSteps = (state: any, validate) => {
```

Comment:
Add type here, too.

It's a pain initially but it forces you to get the types right. (I'm doing something similar in Filings UI.)

Reply:
Added here as well